app/components/draw-control.js

```diff
@@ -82,14 +79,21 @@ export default class DrawControlController extends Component {
   }
 
   @action toggleGeometryEditing(type) {
-    this.set('geometryMode', type);
+    this.set('model.geometryMode', type);
```

Comment:
Yes, I did this to make sure we have a way of knowing which type of geom is currently being drawn to prevent the drawing mode geom and the final geom rendering at the same time.  I'll move it to `project-form`

Reply:
Fixed in 5985043d51b4140747681ff66820d8ff63e69fed